src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AtclauseOrderCheck.java

```diff
@@ -50,7 +50,7 @@
  * Default value is {@code false}.
  * </li>
  * <li>
- * Property {@code target} - Specify the list of targets to check at-clauses.
+ * Property {@code target} - Specify the list of target block tags.
```

Comment:
It is probably better to say this as `block tags targeted`.
I saw some other similar instances.

Reply:
fixed.